BlueShift-iOS-SDK/InApps/BlueShiftNotificationViewController.m

```diff
@@ -399,7 +399,7 @@ - (void)applyIconToLabelView:(UILabel *)iconLabelView andFontIconSize:(NSNumber
             [self createFontFile: iconLabelView];
         }
     
-        CGFloat iconFontSize = (fontSize !=nil && fontSize > [NSNumber numberWithInt:0])? fontSize.floatValue : 22.0;
+        CGFloat iconFontSize = (fontSize && fontSize.doubleValue > 0)? fontSize.floatValue : 22.0;
```

Comment:
why do we need to use `.doubleValue` for comparison and `.floadValue` for return value? Can we use the same code in both places?